documentation/Remove-PnPPlannerBucket.md

```diff
@@ -38,10 +38,10 @@ This cmdlet removes a Planner bucket.
 
 ### Example 1
 ```powershell
-Remove-PnPPlannerBucket -Group "Marketing" -Plan "Conference" -Identity "Preconference Todos"
+Remove-PnPPlannerBucket -Group "Marketing" -Plan "Conference" -Identity "Pre-conference Todo's"
```

Comment:
Maybe we can use `Pre-conf checklist` here ? Also,  todos is the correct plural, so maybe we keep it the same ?

Reply:
agree on keeping same